### PR TITLE
ICDS: Dashboard Fixes

### PR DESCRIPTION
**Pull request by sheelio**
@sravfeyn 

This PR does two things:
- It hides the AWC-only sheets from the dashboard dropdown. This is a bit hard-coded so open to ideas on how to do this better
- It passes location IDs to the dashboard instead of site codes.  This should improve performance since our location tables are indexed on IDs but not on site codes

This will require a corresponding update to the dashboards so we will need to coordinate deploying this.

### CODE REVIEW

File: custom/icds_reports/static/tableau_app.js [hunk 1]
```diff
@@ -54,6 +54,11 @@ function setUpInitialTableauParams() {
 
 function setUpNav(viz) {
     var sheets = workbook.getPublishedSheetsInfo();
+
+    // Filter out the sheets for a single AWC. These are accessed via drilldown
+    sheets = _.filter(sheets, function(sheet) {
+        return (sheet.getName() != "Demographics" && sheet.getName() != 'AWC-Info');
```
Comment (dimagimon) [inline custom/icds_reports/static/tableau_app.js]: Expected '!==' and instead saw '!='. (eqeqeq)

File: custom/icds_reports/static/tableau_app.js [hunk 2]
```diff
@@ -54,6 +54,11 @@ function setUpInitialTableauParams() {
...
+    })
```
Comment (dimagimon) [inline custom/icds_reports/static/tableau_app.js]: Missing semicolon. (semi)

**Comment by czue** [comment]
@sheelio let's try and lump all the dashboard stuff together in case we need to patch it. just ping when ou think you're done with this batch of changes

**Comment by dimagimon** [comment]
There are 6 errors:

* custom/icds_reports/static/tableau_app.js, line 33 - 'currentSheet' is not defined. (no-undef)
* custom/icds_reports/static/tableau_app.js, line 34 - 'currentSheet' is not defined. (no-undef)
'initialLocationParams' is not defined. (no-undef)
* custom/icds_reports/static/tableau_app.js, line 51 - 'initialLocationParams' is not defined. (no-undef)
* custom/icds_reports/static/tableau_app.js, line 52 - 'today' is not defined. (no-undef)
* custom/icds_reports/static/tableau_app.js, line 53 - 'lastMonth' is not defined. (no-undef)
'today' is not defined. (no-undef)
* custom/icds_reports/static/tableau_app.js, line 54 - 'lastMonth' is not defined. (no-undef)

**Comment by czue** [comment]
(build timed out after tests already passed)

**Pull request merged**